### ISSUE TITLE
Grant access of the phone storage to JS/CSS editor

### ISSUE THREAD

**Issue by Palloxin** [LNReader/lnreader#736]
### Describe your suggested feature

Granting access of the phone storage to js/css editor.
### Why
It will allow users to import files from storage.
With CSS one could import new fonts directly from the storage with .tts file.
One could do this
```css
@font-face {
font-family: 'Merriweather';
src: url("file:///storage/3131-3835/Merriweather.ttf");
}
```
but this happens
![image-4](https://github.com/LNReader/lnreader/assets/75091899/adbdf020-2bc1-4362-870d-040264a4ead3)

with Js one could import the content of a js file in the storage without the chore of using the in-app js editor(not suited for code editing).
In this way every time a chapter is loaded the js will be actively updated while the users edits the file using external means (QuickEdit or other apps).
Alas it doesnt work.

### Other details

You could make a toggle option to enable/disable js/css editor storage permissions.
Maybe in the Advanced section of settings

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/LNReader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[1.1.14](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by SkillGG** [comment]
You could try adding [allowFileAccessFromFileURLs and allowUniversalAccessFromFileURLs](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md#allowFileAccessFromFileURLs) flags on the WebView object, which could possibly allow for those to be loaded from storage memory. I can't check it though, because I don't have the space for setting up RN right now.

**Comment by nyagami** [comment]
> You could try adding [allowFileAccessFromFileURLs and allowUniversalAccessFromFileURLs](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md#allowFileAccessFromFileURLs) flags on the WebView object, which could possibly allow for those to be loaded from storage memory. I can't check it though, because I don't have the space for setting up RN right now.

I dont think it could work if you dont give app permission to read the files. Because the js files he write/download must belong to other app (Chrome, File Editor, ...), or I just dont know how to create and edit a new file without using app
